templates/frontend/docker-compose_template.erb

```diff
@@ -49,6 +60,12 @@ services:
     restart: always
     depends_on:
       - haproxy
+<% if @dns.is_a? Array -%>
```

Comment:
se nedan

Reply:
Should be fixed in https://github.com/SUNET/puppet-sunet/pull/12/commits/e729ac21680d98ba6b2d888012959223e5b14467